.github/workflows/upload-release-to-plugin-repo.yml

```diff
@@ -159,3 +206,51 @@ jobs:
                   sed -i "s/Stable tag: ${STABLE_VERSION_REGEX}/Stable tag: ${VERSION}/g" ./readme.txt
                   svn commit -m "Releasing version $VERSION" \
                    --no-auth-cache --non-interactive  --username "$SVN_USERNAME" --password "$SVN_PASSWORD"
+
+    upload-tag:
```

Comment:
How does this workflow get triggered?

Reply:
Are you referring to the entire workflow or this specific job?

The workflow is triggered when a draft release is published (see [#](https://github.com/WordPress/gutenberg/blob/dda3c460c4a5c022ce18883f86fa94f115a9849b/.github/workflows/upload-release-to-plugin-repo.yml#L5)). This specific job only runs if the expression [here](https://github.com/WordPress/gutenberg/pull/49082/files/dda3c460c4a5c022ce18883f86fa94f115a9849b#diff-54a0a47c3496a3f17fce904a8bb01362b517363222210779f7dadb4e8fa2edd3R215-R217) evaluates to true.